xplique/attributions/image_object_detector_explainer.py

```diff
@@ -0,0 +1,214 @@
+"""

```

Comment:
To follow the current linting convention (and match the other files) it would look like this:
```
"""
Module related to Object detector method
"""

from typing import Iterable, Tuple, Union, Optional
import abc

import tensorflow as tf
import numpy as np

from xplique.attributions.base import BlackBoxExplainer
from xplique.commons.tf_operations import inference_batching
```

Reply:
Done.